### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by alexisuwimana**
add files to my repository

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    background: rgba(27, 27, 50, 0.8);
+    border-radius: 5px;
+    box-shadow: 0 0 0.5 1px #ddd;
+}
+
+.data-container4{
+    margin:0 auto 0.45rem auto;
+    color: white;
+
+}
```
Comment (alexisuwimana) [inline css/style.css:108]: **REQUEST FOR CHANGES**

Good job so far your code is clean but if you can make some changes might be done like removing space before the bracket.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+.data-container2{
+    margin: 0 auto 0.45rem auto;
+    color: white;
+}
+.data-container3{
+        margin: 0 auto 0.45rem auto;
+        color: white;
+}
```
Comment (alexisuwimana) [inline css/style.css:65]: you did  a great job but you need to use linters to check you code if are clear na d on standard

File: css/style.css (hunk 1 above)
Comment (GzimAsani) [reply css/style.css:108]: - [ ] Try to rewrite this comment and ask it more politely just like you did in the first comment
- [ ] Use something similar to my template first letting the student know what they did good and then requesting the changes in the comments

**Review by GzimAsani — CHANGES_REQUESTED** [review]
# Hi @alexisuwimana                                     
Good job so far! Your code looks fine, good job :+1:  :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight:
- Good job using inline comments

## Required Changes ♻️ 
- [ ] _Check inline comments for more information :arrow_double_down:_ 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css (hunk 1 above)
Comment (alexisuwimana) [reply css/style.css:108]: **REQUEST FOR CHANGES**

Good job so far your code is clean but if you can make some changes might be done like removing space before the bracket.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        <label for="radio2">Maybe</label>
+        <input type="radio" value='Maybe'   id="radio2" name='user-recommend'>
+
+        <label for="radio3">Not sure</label>
+        <input type="radio" value='Not sure'   id="radio3" name='user-recommend'>
+      </div>
+
+
+      <div class="data-container3">
+
```
Comment (alexisuwimana) [inline index.html:59]: **Change Required**

nice job so far but I suggest you do some changes to remove space between two div in order to have clean code for other

File: css/style.css (hunk 1 above)
Comment (mikethreels) [reply css/style.css:108]: - [ ] a good start on your code review 👍 however as the previous CR mentioned please rewrite the comment and ask it more politely.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Alexis, 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+#survey-form{
+    padding: 1rem;
+    background: rgba(27, 27, 50, 0.8);
+    border-radius: 5px;
+    box-shadow: 0 0 0.5 1px #ddd;
+}
+
+.data-container4{
+    margin:0 auto 0.45rem auto;
+    color: white;
```
Comment (alexisuwimana) [inline css/style.css:106]: **REQUEST FOR CHANGES** 

Good job so far your code is clean but if you can make some changes might be done like removing space before the bracket.

**Comment by helman101** [comment]
# STATUS: APPROVED 🟢 🥳 

Hi @alexisuwimana 🥇 , 

### Congratulations!!! 🥳 🎉

You have successfully completed all the expectations for this project, there is nothing more to do here, now you can go on your way and keep improving your skills.

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**